backends/tofino/bf-p4c/midend/desugar_varbit_extract.h

```diff
@@ -122,9 +123,7 @@ class AnnotateVarbitExtractStates : public Transform {
             if (!method) continue;
 
             if (method->member == "extract" && call->arguments->size() == 2) {
-                IR::Annotations *annotations = state->annotations->clone();
-                annotations->add(new IR::Annotation(IR::ID("dontmerge"), {}));
-                state->annotations = annotations;
+                IR::Annotations::addOrReplace(state->annotations, "dontmerge"_cs, {});
```

Comment:
```suggestion
                state->addAnnotation("dontmerge"_cs, {});
```
Could also work here I believe. Not entirely sure which call is equivalent.